templates/game/game.html.twig

```diff
@@ -42,80 +68,85 @@
-                                            <div class="cell"><div id="cell34"></div></div>
-                                            <div class="cell"><div id="cell35"></div></div>
-                                            <div class="cell obstacle"><div id="cell36"></div></div></div>
-            <div class="row">
-                                            <div class="cell"><div id="cell40"></div></div>
-                                            <div class="cell"><div id="cell41"></div></div>
-                                            <div class="cell obstacle"><div id="cell42"></div></div>
-                                            <div class="cell"><div id="cell43"></div></div>
-                                            <div class="cell obstacle"><div id="cell44"></div></div>
-                                            <div class="cell"><div id="cell45"></div></div>
-                                            <div class="cell"><div id="cell46"></div></div></div>
-            <div class="row">
-                                            <div class="cell"><div id="cell50"></div></div>
-                                            <div class="cell obstacle"><div id="cell51"></div></div>
-                                            <div class="cell"><div id="cell52"></div></div>
-                                            <div class="cell"><div id="cell53"></div></div>
-                                            <div class="cell"><div id="cell54"></div></div>
-                                            <div class="cell obstacle"><div id="cell55"></div></div>
-                                            <div class="cell"><div id="cell56"></div></div></div>
-            <div class="row">
-                                            <div class="cell"><div id="cell60"></div></div>
-                                            <div class="cell"><div id="cell61"></div></div>
-                                            <div class="cell"><div id="cell62"></div></div>
-                                            <div class="cell obstacle"><div id="cell63"></div></div>
-                                            <div class="cell"><div id="cell64"></div></div>
-                                            <div class="cell"><div id="cell65"></div></div>
-                                            <div class="cell"><div id="cell66"></div></div></div>
 
-    <div>
-        <form id="cellactions" method="POST" action="/game">
-            <div>
-                <button name="action" type="submit" value="north">UP</button></div>
-            <div>
-                <button name="action" type="submit" value="west">LEFT</button><button name="action" type="submit" value="east">RIGHT</button></div>
-            <div>
-                <button name="action" type="submit" value="south">DOWN</button></div>
+<form id="cellactions" method="POST" action="/game" class="game">
```

Comment:
There are probably linters for twig! What do you use as IDE?

Reply:
(and there will come a time when I will ask the question about the possibility to manage this grid in a less ... manual way.)

Reply:
I manage it on purpose. The idea was, that I have a fix board, and only the pucks move. The idea is to not render the board each time, it's fixed, but "place" the pucks on it.